### ISSUE TITLE
Support multiple ssh key data by label selector

### ISSUE THREAD

**Issue by craigfurman** [hetznercloud/terraform-provider-hcloud#95]
I would find it useful to be able to retrieve a list of SSH keys by label selector, to add them all to a server.

I would guess that this should be implemented as a separate datasource to `hcloud_ssh_key` to avoid the breaking change? It could be named `hcloud_ssh_keys`. For example, I want to write terraform that looks a bit like the following (TF 0.12 syntax):

```
# Proposed new datasource
data "hcloud_ssh_keys" "prod_ssh" {
  with_selector = "access=prod"
}

resource "hcloud_server" "server" {
  ...
  ssh_keys    = data.hcloud_ssh_keys.keys[*].name
}
```

What do you think? Would you be interested in a PR for this?

**Comment by LKaemmerling** [comment]
This sounds good for me, of course, if you want, you can open a PR with this change :)

**Comment by craigfurman** [comment]
@LKaemmerling great! I added https://github.com/terraform-providers/terraform-provider-hcloud/pull/103 with this change.

**Comment by LKaemmerling** [comment]
Will be released with v1.12.0
